prime-dotnet-webapi/Services/EnrolleeService.cs

```diff
@@ -573,5 +574,25 @@ public async Task<IEnrolleeNote> UpdateEnrolleeNoteAsync(int enrolleeId, IEnroll
 
             return newNote;
         }
+
+
+        public async Task<Enrollee> UpdateEnrolleeAlwaysManualAsync(int enrolleeId, bool alwaysManual)
+        {
+            var enrollee = await _context.Enrollees
+                .Where(e => e.Id == enrolleeId)
+                .SingleOrDefaultAsync();
+
+            enrollee.AlwaysManual = alwaysManual;
+
+            _context.Update(enrollee);
```

Comment:
Is this line needed? I think the changes to enrollee get tracked by the context already